accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -325,6 +354,30 @@ not a subtype of the class type associated with the class in which it is defined
 (specifically, it is an error to return a nullable type from a factory
 constructor for any class other than `Null`).
 
+It is an error if any case of a switch statement except the last case (the
+default case if present) **may complete normally**.  The previous syntactic
+restriction requiring the last statement of each case to be one of a delimited
+set of statements (break, continue, return, throw, or rethrow) is removed.
+
+Given a switch statement which switches over an expression `e` of type `T`,
+where the cases are dispatched based on expressions `e0...ek`:
+  - It is no longer required that the `ei` be instances of the same class.
+  - It is an error if the static type of any of the `ei` is not a subtype of
+    `T`.
+  - It is an error if any of the `ei` evaluate to constants for which equality
+    is not primitive.
+  - If `T` is an enum type, it is a warning if the switch does not handle all
+    enum cases, either explicitly or via a default.
+  - If `T` is `Q?` where `Q` is an enum type, it is a warning if the switch does
+    not handle all enum cases and `null`, either explicitly or via a default.
+  - It is a warning if a switch over a nullable type does not handle `null`
+    either explicitly or via a default.
+
+It is an error if a class as a setter and a getter with the same basename where
+the return type of the setter is not a subtype of the argument type of the
```

Comment:
return type of the setter -> return type of the getter

Reply:
done.